recipe/meta.yaml

```diff
@@ -51,8 +51,8 @@ build:
     - /usr/lib/libresolv.9.dylib  # [osx]
     - /usr/lib/libsandbox.1.dylib  # [osx]
     - '/System/Library/Frameworks/CoreLocation.framework/**'  # [osx]
-    #- '*/api-ms-win-core-winrt-l1-1-0.dll'  # [win]
-    #- '*/api-ms-win-core-winrt-string-l1-1-0.dll'  # [win]
+    - '*/api-ms-win-core-winrt-l1-1-0.dll'  # [win]
+    - '*/api-ms-win-core-winrt-string-l1-1-0.dll'  # [win]
```

Comment:
I think this will prevent qt-webengine from working or being built. I think conda-build doesn' the like leading `*` or something like that.

Reply:
reverted the commit